src/Build/Logging/FileLogger.cs

```diff
@@ -86,6 +86,15 @@ private void InitializeFileLogger(IEventSource eventSource, int nodeCount)
 
             try
             {
+                try
+                {
+                    Directory.CreateDirectory(Path.GetDirectoryName(Path.GetFullPath(_logFileName)));
```

Comment:
Yeah I like that. It should still be caught and logged by the existing mechanism then.